packages/babel-parser/test/helpers/polyfill.js

```diff
@@ -0,0 +1,32 @@
+// We run these tests as far back as Node 6, so we need these there.
```

Comment:
Can you just add a `// TODO(Babel 8): remove this file` comment? So that we don't forget.

Reply:
Sure thing.

Reply:
Done: https://github.com/babel/babel/pull/14201/commits/ffb642e15e3163f14937783ce19a083e58fc74e9